dtfs-central-api/src/services/repositories/durable-functions-repo.ts

```diff
@@ -0,0 +1,16 @@
+import { Collection, DeleteResult } from 'mongodb';
+import { DURABLE_FUNCTIONS_LOG } from '@ukef/dtfs2-common';
+import db from '../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../constants';
+
+const getDurableFunctionsLogCollection = async (): Promise<Collection> => db.getCollection(DB_COLLECTIONS.DURABLE_FUNCTIONS_LOG);
+
+export const deleteAllDurableFunctionLogs = async (): Promise<DeleteResult> => {
+  const durableFunctionLogsCollection = await getDurableFunctionsLogCollection();
+  return durableFunctionLogsCollection.deleteMany({});
+};
+
+export const deleteAllAcbsDurableFunctionLogs = async (): Promise<DeleteResult> => {
+  const durableFunctionLogsCollection = await getDurableFunctionsLogCollection();
+  return durableFunctionLogsCollection.deleteMany({ type: { $eq: DURABLE_FUNCTIONS_LOG.TYPE.ACBS } });
```

Comment:
* Please change this to `$and: [ { type: { $eq: ... } }, { status: 'Completed' } ]`
* Completed to be refered from the constants.